apps/gro/controllers/confirm.js

```diff
@@ -5,6 +5,7 @@ var controllers = require('hof').controllers;
 var BaseController = controllers.base;
 var path = require('path');
 var i18n = require('hof').i18n;
+var _ = require('underscore');
```

Comment:
We need to decide whether we are using underscore or lodash and stick with it. HOF uses lodash but passports use underscore. Personally I prefer lodash as it gives you a lot more than underscore and it's allegedly more efficient too